app/views/schools/cohorts/show.html.erb

```diff
@@ -63,5 +63,15 @@
         </ul>
       </li>
     </ol>
+
+    <% if @school_cohort.core_induction_programme.present? %>
+      <div class="govuk-inset-text">Use these handbooks and training materials to help you plan your school’s
+        <%= govuk_link_to @school_cohort.core_induction_programme.name, induction_tutor_materials_path(
+          provider: @school_cohort.core_induction_programme.name.downcase.tr(" ", "-"), year: :year_one
```

Comment:
`tr(" ", "-")` feels like `dasherize` - can we use that instead?

Reply:
dasherize replaces underscores with hyphens by the looks of it - https://apidock.com/rails/String/dasherize